### PR TITLE
feat: Onboarding script

### PR DESCRIPTION
**Pull request by tbarlow12**
- Added script for generating `.env` file required for running integration tests locally
- Added reference to script in `package.json`
- Updated user utility for asking questions to include a default if desired
- Updated unit tests to reflect new line change

### CODE REVIEW

File: src/scripts/onboard.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import { join } from "path";
+process.env.NODE_CONFIG_DIR = join(__dirname, "..", "config");
```
Comment (elizabethhalper) [inline src/scripts/onboard.ts:2]: Should we clarify in the document name that this is DevOps specific? I'm assuming we'll have a different script for github, so maybe this should be `onboard_devops.ts` or something like that

File: src/scripts/onboard.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import { FileUtils, Logger, UserUtils } from "../utils";
+
+/**
+ * Get the base URL for the Azure DevOps organization
+ *
+ * @returns {Promise<string>} Base URL
+ */
+async function getBaseUrl(): Promise<string> {
+  return UserUtils.askQuestionWithDefault(
+    "What is your Azure DevOps test instance base url?",
```
Comment (elizabethhalper) [inline src/scripts/onboard.ts:12]: I might be misunderstanding the use case here, but from a user perspective, I'm not sure if we should be asking this with a default value. If I'm clicking through and not paying attention and just hitting enter with this default value, I'd have to rerun the script anyway. I'd rather just throw an error or continue asking until they give the url they want to use.

File: src/scripts/onboard.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  );
+}
+
+/**
+ * Get the name for the Azure DevOps project
+ *
+ * @returns {Promise<string>} Project name
+ */
+async function getProjectName(): Promise<string> {
+  return UserUtils.askQuestionWithDefault("What is your Azure DevOps test project name?", "projector-dev");
```
Comment (elizabethhalper) [inline src/scripts/onboard.ts:23]: Same as above comment^

File: src/scripts/onboard.ts (hunk 2 above)
Comment (tbarlow12) [reply src/scripts/onboard.ts:12]: Good point. I'll remove the default

File: src/scripts/onboard.ts (hunk 1 above)
Comment (tbarlow12) [reply src/scripts/onboard.ts:2]: This script should be for all services. The goal is just to create the `.env` file with both GitHub and AzDO info

**Review by elizabethhalper — APPROVED** [review]
Looks good!

**Pull request merged**